lib/uplinkc/main.go

```diff
@@ -0,0 +1,6 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+func main() {}
```

Comment:
is this a cgo thing ?

Reply:
Yes, it requires a `main(){}` for shared library.